edb/edgeql/parser/grammar/statements.py

```diff
@@ -274,12 +274,19 @@ def reduce_empty(self, *kids):
         self.val = False
 
 
-class ExplainStmt(Nonterm):
+class AnalyzeStmt(Nonterm):
 
-    def reduce_EXPLAIN_OptAnalyze_ExprStmt(self, *kids):
+    def reduce_ANALYZE_NamedTuple_ExprStmt(self, *kids):
```

Comment:
Do we want to introduce the explicit `ANALYZE (opts) QUERY` form in 3.0?

Reply:
I don't see why we wouldn't? We've got options that would be useful, so might as well

Reply:
I was confused about the question. My actual answer is "we could, if you want"